### DISCUSSION TITLE
Cant connect

### DISCUSSION THREAD

**Discussion by helvis101** [arkypita/LaserGRBL#1023] [Q&A]
I am having trouble connecting to my Arduino.

I can see my computer recognises that i am connecting it (COM 4)

However, when i hit connect it wont.

What am I doing wrong?

**Comment by arkypita** [comment]
Was the arduino programmed with grbl firmware?

**Comment by helvis101** [comment]
Yeah, I've gone through the instructions to program the grbl firmware. I used Arduino to perform this. I can see there is an option to do this through the LaserGrbl program, however I couldnt get this working. I suspect because I cant connect..

**Comment by Schwarzi07** [comment]
If you are using an Arduino Clone make Sure that you have the driver installed.
